docs/vocabulary.md

```diff
@@ -13,7 +16,7 @@ on a huge amount of text.
 
 #### Masked Language Models (MLMs)
 
-MLMs are a type of NLP model, which have a special token, usually `[MASK]`, which is
+<Term popup="MLMs are a type of NLP model, which have a special token, which is replaced with a word from the vocabulary. The model then predicts the word that was masked." reference="/docs/terms/mlms">MLMs</Term> are a type of NLP model, which have a special token, usually `[MASK]`, which is
```

Comment:
What is the point of the `docs/terms/mlm.md` file if the definition is reused right here?

Reply:
It is removed now

Reply:
If that is the only way to apply it, the md doc will become too verbose. Can you make it so that all instances of MLM are highlighted automatically?